### ISSUE TITLE
Add keyboard navigation support for dropdown filters

### ISSUE THREAD

**Issue by jesustuamigo** [jesustuamigo/trabajo-5F#174]
Dropdown filters are not accessible via keyboard navigation. Implement support for arrow keys and Enter.
